code/espurna/config/progmem.h

```diff
@@ -233,7 +238,9 @@ PROGMEM const char magnitude_pm2dot5_topic[] = "pm2dot5";
 PROGMEM const char magnitude_pm10_topic[] = "pm10";
 PROGMEM const char magnitude_co2_topic[] = "co2";
 PROGMEM const char magnitude_lux_topic[] = "lux";
-PROGMEM const char magnitude_uv_topic[] = "uv";
+PROGMEM const char magnitude_uv_a_topic[] = "uva";
+PROGMEM const char magnitude_uv_b_topic[] = "uvb";
+PROGMEM const char magnitude_uv_index_topic[] = "uv";
```

Comment:
@xoseperez should this be `uvi`? Probably a BC..

Reply:
I have seen it called UVI somewhere, so I guess it's better to say UVI